### DISCUSSION TITLE
usedraggable scrolling fix?

### DISCUSSION THREAD

**Discussion by GradySain** [clauderic/dnd-kit#756] [Q&A] [answered]
I'm new (as of today) to dnd-kit so bear with me. I'm using dnd-kit/sortable, and I'm experiencing the behavior illustrated here:
https://github.com/clauderic/dnd-kit/pull/660
in the first "before" video.  I'm wondering how to employ the fix as illustrated, using "useDraggable".

**Comment by clauderic** [comment, marked as answer]
**Edit**: It has now been released as of `@dnd-kit/core^6.0.0`

**Original answer**:
The fix isn't in a major release yet (soon!). You can install @dnd-kit packages tagged with ˋnext` on npm to use the latest experimental release that includes the fix you are referring to.

ˋnpm install @dnd-kit/core@next`

**Reply by GradySain** [reply]
oh! thanks very much i will try it out. crazy nice library BTW congrats

**Comment by GradySain** [comment]
it worked out of the box, great!
